app/src/main/java/com/example/leagueoflegends/presenter/MainActivityPresenter.kt

```diff
@@ -0,0 +1,25 @@
+package com.example.leagueoflegends.presenter
+
+import com.example.leagueoflegends.contract.BaseContract
+import com.example.leagueoflegends.misc.ApiService
+import com.example.leagueoflegends.misc.RetrofitProvider
+import com.example.leagueoflegends.model.Response
+import io.reactivex.Single
+
+class MainActivityPresenter: BaseContract.BasePresenter {
```

Comment:
you can just call this main presenter, don't have to specify activity, because maybe later you'll change it into a fragment or something else